### DISCUSSION TITLE
Can't find model 'en_core_web_md'.

### DISCUSSION THREAD

**Discussion by rahul-art** [explosion/spaCy#7364] [Help: Installation] [answered]
## How to reproduce the behaviour
![Screenshot from 2021-01-23 05-18-08](https://user-images.githubusercontent.com/56379013/105560658-778abf00-5d3a-11eb-9cc8-ed19d1120510.png)

## Your Environment
ubuntu 
## Info about spaCy

* **spaCy version:** 2.2.4
* **Platform:** Linux-4.19.112+-x86_64-with-Ubuntu-18.04-bionic
* **Python version:** 3.6.9
* **Models:** en

**Comment by svlandeg** [comment, marked as answer]
It looks like you're not actually downloading the `md` and `lg` models:

![image](https://user-images.githubusercontent.com/8796347/105681366-56b89a00-5ef1-11eb-8ee9-5b0eeb96973d.png)
